src/xcode/ENA/ENA/Source/Views/BaseElements/ENALinkedTextView.swift

```diff
@@ -0,0 +1,85 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import UIKit
+
+class ENALinkedTextView: UITextView {
```

Comment:
Is this really needed? I think @KaiTeuber mentioned something that we already have something like this? 🤔

Reply:
I extracted/copied the solution from Kai into this component to make it reusable for my case.